code/modules/clothing/clothing.dm

```diff
@@ -217,6 +217,27 @@
 		for(var/A in armor)
 			armor[A] -= rand(armor[A]/3, armor[A])
 
+// Redefines the attack proc so that when we attack somebody on fire on help intent with a piece of clothing, we have a small chance of putting out the fire.
+/obj/item/clothing/attack(mob/living/M, mob/living/user, def_zone, var/originator = null)
+	if(iscarbon(user)) // First, we check if it's a a carbon.
```

Comment:
Comments like these are just clutter, in my opinion. Comments should explain _why_ you're doing something; in cases like these where _what_ you're doing is self-explanatory from the code alone they are not needed. I'd say keep the one explaining why the plasmamen are excluded but get rid of this one, the help intent one, the on_fire one, the polymorphism one.

Reply:
>self-explanatory
I disagree with that, most of the lines are relatively obscure in my opinion.
But again, I've been trained to put comments basically everywhere so

Reply:
Read this line of code again and tell me the comment adds any information that was not already clear